test/rubygems/test_gem_config_file.rb

```diff
@@ -183,7 +183,7 @@ def test_api_keys
   end
 
   def test_check_credentials_permissions
-    skip 'chmod not supported' if win_platform?
+    skip 'chmod 0644 not supported' if win_platform?
```

Comment:
Hm, I think the real issue is that 0600 is not supported, isn't it?

Reply:
I don't work that much with chmod.  Would it be better to change `0644` to `06xx`?

FYI, the test for case insensitive should use `File.identical?`, as it would currently pass in case sensitive file systems if one actually had directories like `/c/RubyGems` and  `/c/rubygems`.  Unlikely, but possible.  Pointed out by nobu.  I will change that...

Reply:
Well, chmod on Window supports two modes - 0644 and 0444. All it really does is set the "read-only" flag on file properties. You can see it if you right click on a file and select "properties". It's basically a weak attempt at POSIX compatibility.

My main point is that, according to the error message at least, the rubygems config file needs to be 0600, but that's not supported.

There's other ways to set security on Windows files (see e.g. win32-security and/or win32-file-attributes), but we'd have to establish what the rules should be.